build.gradle

```diff
@@ -1,14 +1,14 @@
 plugins {
     id 'java'
     id 'maven-publish'
-    id "ca.coglinc2.javacc" version "3.0.0"
+    id "ca.coglinc2.javacc" version "latest.release"
     id 'jacoco'
-    id "com.github.spotbugs" version "4.7.2"
+    id "com.github.spotbugs" version "latest.release"
     id 'pmd'
     id 'checkstyle'
     
     // download the RR tools which have no Maven Repository
-    id "de.undercouch.download" version "4.1.2"
+    id "de.undercouch.download" version "latest.release"
```

Comment:
I do not know about Maven does (In fact I hate Maven. You can't even `clean` a project while offline. I went straight from ANT to Gradle, which "just understands and does what I want" instead of forcing me into any dogma). This `latest.release` automatically retrieves the Latest Release of a Gradle Plugin.